### DISCUSSION TITLE
Remove pre-CSS code

### DISCUSSION THREAD

**Discussion by buhtz** [pygments/pygments#2395] [Q&A] [answered]
The first lines of my CSS file do look like this

```
pre { line-height: 125%; margin: 0; }
td.linenos pre { color: #000000; background-color: #f0f0f0; padding: 0 5px 0 5px; }
span.linenos { color: #000000; background-color: #f0f0f0; padding: 0 5px 0 5px; }
td.linenos pre.special { color: #000000; background-color: #ffffc0; padding: 0 5px 0 5px; }
span.linenos.special { color: #000000; background-color: #ffffc0; padding: 0 5px 0 5px; }
pre .hll { background-color: #ffffcc }
pre { background: #f8f8f8; }
pre .c { color: #408080; font-style: italic } /* Comment */
pre .err { border: 1px solid #FF0000 } /* Error */
pre .k { color: #008000; font-weight: bold } /* Keyword */
```

I would like to remove the lines modifying `<pre>` because they pollute my own pre-css-class.

```
pre { line-height: 125%; margin: 0; }
pre { background: #f8f8f8; }

```

**Comment by kurtmckee** [comment, marked as answer]
Would it be possible to specify your own CSS file after the Pygments CSS file, and override the `line-height`, `margin`, and `background` values?

**Reply by buhtz** [reply]
Hey Kurt! 😃 
Yes it is a good workaround. In the end I used another solution. I just [removed the relevant lines](https://codeberg.org/buhtz/hyperorg/src/commit/5f0e21a537ebdc27affa78678194e4860f395dbc/src/hyperorg/exporter.py#L181) (all starting with `pre { `) from `pygments.css` file.

**Reply by kurtmckee** [reply]
Super! :grin:
